src/main/java/com/marklogic/kafka/connect/source/RowBatcherSourceTask.java

```diff
@@ -57,12 +57,15 @@ public final void start(Map<String, String> config) {
         this.databaseClient = new DefaultConfiguredDatabaseClientFactory().newDatabaseClient(databaseClientConfig);
         dataMovementManager = databaseClient.newDataMovementManager();
         pollDelayMs = (Long) parsedConfig.get(MarkLogicSourceConfig.WAIT_TIME);
-        this.constraintColumn = (String) parsedConfig.get(MarkLogicSourceConfig.CONSTRAINT_COLUMN_NAME);
+        constraintColumn = (String) parsedConfig.get(MarkLogicSourceConfig.CONSTRAINT_COLUMN_NAME);
         if (!StringUtils.hasText(constraintColumn)) {
             constraintColumn = null;
+        } else {
+            constraintColumn = constraintColumn.replaceAll("[\"']", "");
```

Comment:
This line of code is repeated in DslConstraintColumnHandler, which means it's a good candidate to be stuffed into a public static "util" method somewhere where it can also be easily unit-tested to cover what it's intended to handle.